lib/xsl/jats-to-html.xsl

```diff
@@ -1105,6 +1105,11 @@
                     <xsl:value-of select="concat('table-', @align)"/>
                 </xsl:attribute>
             </xsl:if>
+            <xsl:if test="@style">
+                <xsl:attribute name="class">

```

Comment:
Should be `<xsl:attribute name="style">` if we're letting the styles come through as it.